main-command/src/main/scala/sbt/BasicKeys.scala

```diff
@@ -123,6 +123,12 @@ object BasicKeys {
     10000
   )
 
+  val localDigestCacheByteSize = AttributeKey[Long](
+    "localDigestCacheByteSize",
+    "The maximum total size in the in-memory digest cache in bytes.",
+    10000
```

Comment:
@eed3si9n In the PR description and the commit message it's said that the default cache limit is set to 1MB, but looks like it's actually ~10KB. Is that intentional? 🤔

Reply:
The third number of `AttributeKey.apply` is `rank`, which is used to show/hide settings in help command. The actual initialization is done in RemoteCache.scala as:

```scala
localDigestCacheByteSize :== CacheImplicits.defaultLocalDigestCacheByteSize,
```

which in turn is set to

```scala
  private[sbt] val defaultLocalDigestCacheByteSize = 1024L * 1024L
```